### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by mddo6191**
this is my new feautre1. could you plz check it. thanks

### CODE REVIEW

File: src/feature1.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+print ("this is feature 1")
+
+
```
Comment (nguacon01) [inline src/feature1.py]: you need to delete these whitespaces

**Review by nguacon01 — COMMENTED** [review]
also

File: src/feature1.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+print ("this is feature 1")
+
```
Comment (nguacon01) [inline src/feature1.py]: you need to delete whitespace

Comment (mddo6191) [reply src/feature1.py]: whitespaces are already deleted

**Review by nguacon01 — APPROVED** [review]

**Pull request merged**